src/partials/modal.html

```diff
@@ -0,0 +1,48 @@
+<div class="backdrop is-hidden" data-modal>
+  <div class="modal">
+    <div class="modal-top">
+      <button
+        type="button"
+        class="btn-burger header-icon-cross"
+        aria-expanded="false"
+        data-modal-close
+      >
+        <svg
+          class="header-icon-cross"
+          width="20"
+          height="20"
+          aria-label="Переклчатель мобильного меню"
+        >
+          <use class="header-icon-cross" xlink:href="../images/sprite.svg#cross"></use>
+        </svg>
+      </button>
+      <a href="./index.html" class="logo-modal"
+        ><img
+          class="logo-modal"
+          src="../images/logo-modal.png"
```

Comment:
Нужно заменить 2 точки на 1, как сделано на примере снизу:

```suggestion
          src="./images/logo-modal.png"
```